pkg/storage/stores/shipper/downloads/table_manager.go

```diff
@@ -0,0 +1,172 @@
+package downloads
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/cortexproject/cortex/pkg/chunk"
+	chunk_util "github.com/cortexproject/cortex/pkg/chunk/util"
+	pkg_util "github.com/cortexproject/cortex/pkg/util"
+	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const cacheCleanupInterval = 24 * time.Hour
+
+type Config struct {
+	CacheDir     string
```

Comment:
Should we support flags too ? Usually nice for documenting fields or overriding. Not sure @slim-bean ?

Reply:
Ho just realized this is passed down another configuration that has flags support. Nevermind.